### ISSUE TITLE
Dropout: a simple way to prevent Neural Networks from overfitting

### ISSUE THREAD

**Issue by TanUkkii007** [TanUkkii007/papers-i-read#53]
link: https://www.cs.toronto.edu/~hinton/absps/JMLRdropout.pdf

referenced from:
- Convolutional Sequence to Sequence Learning #50 for variance scaling effect by dropout that requires careful weight initialization
